lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -6,7 +6,7 @@ class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmC
     expose :resize_confirm, :override_return => nil
     expose :resize_revert,  :override_return => nil
 
-    expose :validate_resize
+    expose :supports_resize?
```

Comment:
@durandom That's the nice thing about automate, you never really know if/when the methods are used since it is referenced from user code.  😦   This is pretty new stuff (PR #7922 was merged in April), I do not know of anyone using it.

The suggested solution of adding the `validate_resize` method would work.

Note: If it helps you can also rename the backend method and redirect automate to the name method using `:method => <new_method_name>`.
[lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb#L19](https://github.com/ManageIQ/manageiq/blob/51ed1eb89aecf2c0481ca7953cdd7df1c6de691a/lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb#L19)